docs/services/hpc-console.md

```diff
@@ -0,0 +1,53 @@
+[](){#hpc-console}
+# HPC Console
+
+HCP Console is a web application designed to provide essential functionalities for interacting with [FirecREST API v2 backend][ref-firecrest].
+It provides users with a modern web-based environment to interact with HPC resources such as clusters, job schedulers, and filesystems.
+
+## Features
+
+- A modern web interface built on FirecREST API functionalities.
+- A dashboard offering an overview of configured clusters and their statuses.
+- A dedicated view for creating and monitoring jobs.
+- A file navigator with basic file management functionalities, including upload and download features.
+
+A description of the views and functionalities can be found [here](https://eth-cscs.github.io/firecrest-ui/documentation/).
+
+## Views overview
+
+- Dashboard – shows cluster availability and basic system status.
+- Jobs – allows creating, submitting, and monitoring jobs. Users can provide job name, script, and resource requirements.
+- File Navigator – supports browsing directories, uploading, downloading, and deleting files. Editing files inline is not supported.
+
+## Supported version
+
+Starting early 2025, CSCS has introduced a new version of the API: [FirecREST version 2](https://eth-cscs.github.io/firecrest-v2).
+
+This web interface is available and runs on the FirecREST API v2 backend, FirecREST API v1 is not supported.
```

Comment:
One thing that's not quite clear to me from this document (given the split between "HPC console" and "firecrest UI" links): is this page documentation about the particular _instance_ of firecrest UI hosted at CSCS for CSCS clusters, or is this a description of firecrest UI (with a renaming in flux)? If the former, I'm not sure if a user cares here about which version of firecrest is being used.

However, I may very well be missing some nuance as to why it's useful to mentionn this here.